code/modules/projectiles/projectile/energy.dm

```diff
@@ -21,11 +19,13 @@
 
 /obj/item/projectile/energy/electrode/on_hit(var/atom/target, var/blocked = 0)
 	. = ..()
+	var/mob/living/carbon/C = target
```

Comment:
Why did you move this up here when it's possible `target` is not carbon?

Reply:
uhhhh...... will fix